logging/logs.js

```diff
@@ -13,114 +13,136 @@
-  }
+function writeLogEntry (logName, callback) {
+  var logging = Logging();
+  var log = logging.log(logName);
+
+  // Modify this resource to match a resource in your project
+  // See https://cloud.google.com/logging/docs/api/ref_v2beta1/rest/v2beta1/MonitoredResource
+  var resource = {
+    // This example targets the "global" resource for simplicity
+    type: 'global'
+  };
+
+  // A text log entry
+  var entry = log.entry(resource, 'Hello, world!');
+
+  // A structured log entry
+  var secondEntry = log.entry(resource, {
+    name: 'King Arthur',
+    quest: 'Find the Holy Grail',
+    favorite_color: 'Blue'
+  });
 
-  // See https://googlecloudplatform.github.io/gcloud-node/#/docs/logging/latest/logging
-  logging.getEntries(config, function (err, entries) {
+  // Save the two log entries. You can write entries one at a time, but it is
+  // best to write multiple entires together in a batch.
+  log.write([
+    entry,
+    secondEntry
+  ], function (err, apiResponse) {
     if (err) {
       return callback(err);
     }
 
-    console.log('Found %d entries!', entries.length);
-    return callback(null, entries);
+    console.log('Wrote to %s', logName);
+    return callback(null, apiResponse);
   });
 }
-// [END list_log_entries]
-
-// [START write_log_entry]
-/**
- * Write a log entry.
- *
- * @param {object} options Configuration options.
- * @param {string} options.name The name of the log to write to.
- * @param {object} options.resource The resource to associate with the log entry.
- * @param {string|object} options.entry The body of the log entry.
- * @param {function} callback The callback function.
- */
-function writeLogEntry (options, callback) {
-  // Get a reference to an existing log
-  var log = logging.log(options.name);
+
+function writeLogEntryAdvanced (logName, options, callback) {
+  var logging = Logging();
+  var log = logging.log(logName);
 
   // Prepare the entry
   var entry = log.entry(options.resource, options.entry);
 
-  // See https://googlecloudplatform.github.io/gcloud-node/#/docs/logging/latest/logging/log
-  log.write(entry, function (err) {
+  // See https://googlecloudplatform.github.io/google-cloud-node/#/docs/logging/latest/logging/log?method=write
+  log.write(entry, function (err, apiResponse) {
+    if (err) {
+      return callback(err);
+    }
+
+    console.log('Wrote entry to log: %s', logName);
+    return callback(null, apiResponse);
+  });
+}
+
+function listLogEntries (logName, callback) {
+  var logging = Logging();
+  var log = logging.log(logName);
+
+  // List the most recent entries for a given log
+  // See https://googlecloudplatform.github.io/google-cloud-node/#/docs/logging/latest/logging?method=getEntries
+  log.getEntries(function (err, entries) {
+    if (err) {
+      return callback(err);
+    }
+
+    console.log('Found %d entries!', entries.length);
+    return callback(null, entries);
+  });
+}
+
+function listLogEntriesAdvanced (filter, limit, sort, callback) {
```

Comment:
Is there any benefit to making these parameter names the same as their respective `options` values?